pandas/tests/dtypes/test_inference.py

```diff
@@ -62,46 +62,46 @@ def coerce(request):
 # collect all objects to be tested for list-like-ness; use tuples of objects,
 # whether they are list-like or not (special casing for sets), and their ID
 ll_params = [
-    ([1], True, "list"),  # noqa: E241
-    ([], True, "list-empty"),  # noqa: E241
-    ((1,), True, "tuple"),  # noqa: E241
```

Comment:
@kaathi524 what's the reason this is not necessary any more?

Reply:
@jorisvandenbossche Yeah. According to Issue #29607 , Need to remove those noqa.

Reply:
OK, thanks!